backend/src/controllers/items-controller.ts

```diff
@@ -122,7 +134,7 @@ export const updateItem = async (req: CustomRequest, res: Response) => {
     //Updates item
     try {
         const result = await Item.updateOne({_id: new ObjectId(itemID)},
-            { name: newName, description: newDescription, pickupLocation: newPickupLocation, pickupTime: newPickupTime });
+            { name: newName, description: newDescription, pickupLocation: newPickupLocation, pickupTime: newPickupTime, groupId: groupID });
```

Comment:
Before adding the `groupId` to the `Item`, check that a `Group` object exists with its `_id` equal to `groupId`